peacebot/core/plugins/Miscellaneous/__init__.py

```diff
@@ -1,17 +1,9 @@
 import lightbulb
 from apscheduler.schedulers.asyncio import AsyncIOScheduler
 
-from peacebot.core.utils.time import TimeConverter
-
 
 def fetch_scheduler(ctx: lightbulb.Context) -> AsyncIOScheduler:
-    return ctx.bot.d.scheduler
-
-
-async def convert_time(ctx: lightbulb.Context, time: str) -> float:
```

Comment:
Why did we remove the convert time function?

Reply:
I have relocated the `convert_time` to `peacebot/core/utils/time.py`